packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -136,6 +143,15 @@ const useDevtoolsStyles = makeStyles({
 	},
 });
 
+/**
+ * Telemetry base logger passed from the {@link DevtoolsPanel}.
+ * Passed in to {@link DevtoolsView} since it receives the {@link DevtoolsFeatures.Message}.
+ *
+ */
```

Comment:
This can go in the `usageTelemetryLogger` property directly; for the whole props object we can do something a bit more generic, look at the props for other components.